appvm-scripts/etc/init.d/qubes-gui-agent

```diff
@@ -12,7 +12,7 @@ start()
 	# start console-kit-daemon
 	/usr/bin/ck-list-sessions > /dev/null 2>&1
 	# pretend tha user is at local console
-	touch /var/run/console/user
+	touch /run/console/user
```

Comment:
As discussed in https://github.com/QubesOS/qubes-issues/issues/6315#issuecomment-752059136, this should stay /var/run.